swan/scripts/before-notebook.d/01_storage.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Author: Danilo Piparo, Enric Tejedor, Pedro Maximino, Diogo Castro 2023
+# Copyright CERN
+# Here the storage environment is prepared, to ensure the user is able to access
+# their home directory (EOS for CERN users).
+
+# The $HOME directory is specified upstream in the Spawner
+if [[ $HOME == /eos/* ]]; then export CERNBOX_HOME=$HOME; fi
```

Comment:
Is CERNBOX_HOME strictly needed? I think now we use it in the user environment script box as an example, but could we just get rid of it? At the very least, if we should not export it.

Reply:
If we advertise that as the ENV var that points to CERBox (including in the startup form), I would expect users to use it.